samples/charts/financial-chart/annotations/Pages/FinancialChartAnnotations.razor

```diff
@@ -0,0 +1,182 @@
+﻿@page "/CategoryChartOverview"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+@inject Samples.Services.StocksUtility StocksUtility;
+@inject Samples.Services.Stock Stock;
```

Comment:
remove these lines because they are not needed:
`@inject Samples.Services.StocksUtility StocksUtility;`
`@inject Samples.Services.Stock Stock;`
see my comments in StocksUtility.cs